### ISSUE TITLE
Airline theme breaks with transparent vim windows.

### ISSUE THREAD

**Issue by ry-dgel** [morhetz/gruvbox#108]
I'm using Urxvt with real transparency and Compton as a compositor. In order to get a transparent background with nvim I set, `:hi Normal ctermbg=none` in init.vim.

This causes a bunch of red to take over in vim-airline, as it gets the colors from the theme after the previous change is made. Without changing my init.vim, if I create a new window and then pass the previous statement as a command, everything works fine, as vim-airline has already read the right colors.

Before.
![2016-03-05-221500_931x43_scrot](https://cloud.githubusercontent.com/assets/14968322/13552075/ce89ffc2-e31f-11e5-8e4a-4d8507ee7af0.png)

After. (Statement in config.)
![2016-03-05-221433_923x40_scrot](https://cloud.githubusercontent.com/assets/14968322/13552076/ce8a0a30-e31f-11e5-9d47-0c350e40cae2.png)

How it should look. (Statement passed as command in existing window.)
![2016-03-05-221518_925x41_scrot](https://cloud.githubusercontent.com/assets/14968322/13552074/ce89980c-e31f-11e5-8840-7737e2ff48ee.png)

**Comment by ry-dgel** [comment]
I found a work around for now, by changing the background using a vim autocmd, the change only takes effect after everything loads.

`autocmd VimEnter * hi Normal ctermbg=none`

Does this perfectly.

**Comment by alpertuna** [comment]
@ry-dgel I experienced similar problem and your solution works well, thanks

**Comment by 3ximus** [comment]
I was having this issue either with @ry-dgel autocmd Solution an without.
I'm running `let g:gruvbox_termcolors = 16` and I guess this sets`highlight Normal ctermfg=15 ctermbg=0` and when reversed in the airline theme it uses the bold version of this color (8) for the mode foreground resulting in these:

![print_003](https://user-images.githubusercontent.com/9083012/100171968-37849880-2ebf-11eb-98f1-27bf36f31a11.png)
![print_001](https://user-images.githubusercontent.com/9083012/100171914-1b80f700-2ebf-11eb-8de2-0b272ff80d84.png)

So for me the solution was to define a different color for the background, like so:
```hi Normal ctermbg=236```
This will make the bold color be 236 too and now works fine.
![print_004](https://user-images.githubusercontent.com/9083012/100172268-d6a99000-2ebf-11eb-9e6a-6db1befeff05.png)

**Comment by normen** [comment]
I am having the same issue when enabling transparency. The `autocmd` solution works somewhat but when I open a `:term` it reverts back to the old behavior of displaying wrong colors.

**Comment by 3ximus** [comment]
@normen Do you by any chance use the terminal for fzf only ? That was my case and for that I added an option to fzf.vim plugin to prevent it from using the terminal when available

**Comment by normen** [comment]
No, I am using a zsh with only a few options. I moved from airline to lightline and that seemed to fix the issue, plus some others 😄
